esp/platform/espcontext.cpp

```diff
@@ -172,7 +173,7 @@ class CEspContext : public CInterface, implements IEspContext
     }
     virtual const MemoryBuffer & querySessionToken()
     {
-        return m_user->credentials().getSessionToken();
+        return m_user ? m_user->credentials().getSessionToken() : m_emptyMB;
```

Comment:
The token and signature are binary, hence the use of MemoryBuffer instead of "char *" and StringBuffer.  I suppose I could entertain returning a "const MemoryBuffer *", would that be okay?